types/office-js/index.d.ts

```diff
@@ -69688,22 +69416,21 @@ declare namespace Word {
      * ContentControl appearance
      *
      * [Api set: WordApi]
+     * @remarks
```

Comment:
These remarks don't seem to be useful... (may also be applicable to office-js-preview)